agents/kubevirt/fence_kubevirt.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python -tt
+
+import os
+import sys, re
+import logging
+import atexit
+sys.path.append("/usr/share/fence/")
+from fencing import *
+from fencing import fail, fail_usage, run_delay, EC_STATUS, SyslogLibHandler
+from kubernetes import client, config
+from openshift.dynamic import DynamicClient
+from kubernetes.client.exceptions import ApiException
+
+API_VERSION='kubevirt.io/v1'
+
+logger = logging.getLogger("fence_kubevirt")
+logger.propagate = False
+logger.setLevel(logging.INFO)
+logger.addHandler(SyslogLibHandler())
+logging.getLogger('botocore.vendored').propagate = False
+
+def get_nodes_list(conn, options):
+    logger.info("Starting monitor operation")
```

Comment:
You should use debug loglevel here (and maybe say "list/monitor" instead), so it doesnt spam the logs everytime the monitor action is run.

Reply:
Fixed